quickstarts/microsoft.storage/storage-blob-encryption-and-retention/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "storageAccountName": {
+            "value": "GEN-UNIQUE"
+        },
+        "storageSku": {
```

Comment:
move the literal param values to defaultValues in the template... that will make it easier to deploy via portal.

Reply:
@bmoore-msft moved param values to default values as requested